amplify/backend/backend-config.json

```
@@ -60,6 +60,22 @@
     }
   },
   "parameters": {
+    "AMPLIFY_function_cognitoCreateNetpuneUser_deploymentBucketName": {
+      "usedBy": [
+        {
+          "category": "function",
+          "resourceName": "cognitoCreateNetpuneUser"
```

Might have to rename the resource
```suggestion
          "resourceName": "cognitoCreateNeptuneUser"
```

---

we're actually just going to use postUser. This function will be deleted.

---

I'm not sure why this is still there, the function is deleted from my branch.